pkg/systemd/terminal.html

```diff
@@ -7,23 +7,10 @@
     <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
     <script src="../base1/bundle.js"></script>
     <script src="../system/bundle.js"></script>
-<script>
-    require([
-        "jquery",
-        "system/terminal"
-    ], function($, terminal) {
-        $(terminal);
-    });
-</script>
+    <script type="text/javascript" src="../base1/cockpit-components.js"></script>
+    <script src="terminal.js"></script>
```

Comment:
I think you're including this twice.
I recommend:
```
    <!-- debug:start -->
    <script src="terminal.js"></script>
    <!-- debug:end -->
```
The file might not even be present when installing via packages.